userdocs/profiles.dev/docs/tutorial-basics/create-a-profile.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 2
+---
+
+# Write a profile
+
+:::info
+
+This stage of the tutorial assumes you have prepared your environment correctly.
+
+Please refer back to the [previous section](/docs/tutorial-basics/setup) if not.
+:::
+
+In this section we are going to write a very simple profile.
+
+_If you do not wish to learn about writing profiles, please skip ahead to the next section._
+
+---------------------
+
+You will need an account with a git provider (eg GitHub or GitLab) and a text editor.
+
+We are going to write a profile which can be used to install a single component: nginx.
+Yes, I know it is not the most exciting thing, but it serves us well for a quick illustration.
+
+Create a new repository:
+```bash
+mkdir demo-profile
+cd demo-profile
+git init
+# other git setup things, etc
+```
+
+At a bare minimum a profile only needs to have one thing in it: a `profile.yaml` file.
+In here we define object Kind and the profile components, known as `artifacts`, under the spec.
+
+```yaml
+apiVersion: weave.works/v1alpha1
+kind: ProfileDefinition
+metadata:
+  name: nginx
+spec:
```

Comment:
```suggestion
spec:
  description: Profile for deploying the bitnami nginx chart
  maintainer: Aunty Ethel
  prerequisites:
    - "kubernetes 1.19"
```

Reply:
are those required fields?

Reply:
"required" no, recommended yes, they get displayed in the catalog.

Reply:
they will be in another part of the docs. getting started tutorials only contain enough to get started and catalogs are not needed yet.